src-env (msdb)/.env/dev-use1/.env

```diff
@@ -1,23 +1,30 @@
-REGION=USE1 #East USA 1
-ENVIRONMENT=DEV
-SERVICENAME=_microservicetemplate_-server
-SERVICE=MSTS
+Region=USE1 #East USA 1
+Environment=DEV
+ServiceName=_microservicetemplate_-server
+Service=MSTS
 
 AllowedHosts=*
 DisableSwagger=false
+InjectDBCredentialFromEnvironment=false -------- obviouse!!!
```

Comment:
rempve the " -------- obviouse!!!" 
it is false or true, and default should be true, move it above ASPNETCORE_DB_USERNAME